### ISSUE TITLE
WebView gets a strange brown border upon hiding / showing a window

### ISSUE THREAD

**Issue by MarshallOfSound** [atom/electron#4768]
* Electron version: `0.37.0`
* Operating system: Windows 10 x64

As seen here, a brown border appears around the webview immediately after you hide and show the browser window.  It then disappears on ANY interaction with the window, you just need to click anything once and it disappears.

![orange-border](https://cloud.githubusercontent.com/assets/6634592/13723490/65243d6c-e8bb-11e5-8757-125cb457f03a.PNG)

I'm 99.99999% sure it isn't any stray CSS on my side, hence raising it here.

**Comment by bezformy** [comment]
Seems like the behavior of the `outline` css property. What happens if you set `outline:none` on it?

**Comment by paulcbetts** [comment]
This is the focus marker, set the focus to the `<object>` tag inside the Webview and it'll go away

**Comment by zcbenz** [comment]
Closing since it is a CSS issue.

**Comment by akashnimare** [comment]
I'm getting the same border. I'm not sure if this is a CSS issue. Resizing window fixes it. I used the following code as suggested by @MarshallOfSound but it didn't work -

```javascript
webview::shadow object {
    height: 100%;
    width: 100%;
    outline: 0 !important;
    outline-offset: 0 !important;
  }
```
<img width="578" alt="screen shot 2018-03-11 at 7 03 32 pm" src="https://user-images.githubusercontent.com/2263909/37254305-0d76bdfc-2562-11e8-81f2-b898e7cf8ba6.png">

(electron v1.8.2)

(I changed the background to red here)
